conjur/cli.py

```diff
@@ -561,7 +570,7 @@ def handle_logout_logic(cls, ssl_verify=True):
         """
         Method that wraps the logout call logic
         """
-        credentials = CredentialsFromFile(DEFAULT_NETRC_FILE)
+        credentials = NetrcWrapper(DEFAULT_NETRC_FILE)
```

Comment:
Purposefully didn't want to name is Netrc-something. B/c today it is netrc, sometomorrow it something else

Reply:
I am also not sure why it is a wrapper? What is your thoughts here

Reply:
But it has function build_netrc. not build_configfile. so we need to change the function name also ?

Reply:
I agree with Sigal, and with your addition :)
We should abstract any external resource and use business phrases to describe the action